public/coffee/ide.coffee

```diff
@@ -72,13 +72,13 @@ define [
 		# Tracking code.
 		$scope.$watch "ui.view", (newView, oldView) ->
 			if newView? and newView != "editor" and newView != "pdf"
-				event_tracking.sendCountly "ide-open-view-#{ newView }" 
+				event_tracking.sendCountlyOnce "ide-open-view-#{ newView }" 
```

Comment:
shall we change these event names to something like "has-done-ide-open-view" or "onetime-ide-open-view"? open for debate.